eng/common/tools.sh

```diff
@@ -477,6 +477,11 @@ function MSBuild {
     args+=( "-logger:$selectedPath" )
   fi
 
+  # Workaround for a bug where .editorconfig files in nuget packages are not respected if the NUGET_PACKAGES path does not end with a slash
```

Comment:
This is insance I never know how you smart people find these wacky edge cases. Truly impressed 😲

Reply:
Agreed 💯! @dibarbet routinely blows my mind. :smile:

Reply:
I can figure this out but can't figure out how to call a shell script in CI 😢